interview-questions.md

```diff
@@ -7,48 +7,55 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is an instance variable in Ruby? How is it different from other variables?
 
-  Your answer:
+  Your answer: The syntax for an instance variable is @ followed by the variables assigned name. This is (as the name reveals) is an instance of the chosen variable and not a direct call of the variable itself. 
 
-  Researched answer:
+  Researched answer: After digging a little deeper I learned that what follows the instance variable while its being assigned is actually it's scope. Also i discovered that instance variables are not global.
 
 
 
 2. What is a block in Ruby?
 
-  Your answer:
+  Your answer: Blocks in ruby are built in methods that require a do/end
 
-  Researched answer:
+  Researched answer: After researching blocks a little more I discovered that blocks are anonymous pieces of code that can also use curly braces {} instead of do/end
 
 
 
 3. Ruby has an implicit return. What does that mean?
 
-  Your answer:
+  Your answer: This means that the last line of code that you write in a program will automatically be attempted to be returned by Ruby
 
-  Researched answer:
+  Researched answer: To improve on my previous definition, I would add that an implict return is automatically called upon the last executed instruction in a method when an explicit return has not been used. 
 
 
 
 4. What is object-oriented programming? How is it different than functional programming?
 
-  Your answer:
+  Your answer: This is a programming mind set that focuses on the functionality of objects and thus builds around that functionailty 
 
-  Researched answer:
+  Researched answer: The main difference between the two is that functional programming cannot store data in objects and depends upon the creation of functions to do so. This compaired to object-orients programming which can store data in objects
 
 
 
 5. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: A class stores key words that are tied to objects
```

Comment:
Pretty decent answer. Again, i'd encourage you to really consider the question and if your answer covers both aspects of the question. 

Let's think of the question this way: If someone asked you "Whats the difference between a plate and a table?"
I wouldn't want to reply: "A plate goes onto a table." 
A more appropriate reply would be, perhaps, "A table is a piece of furniture that can hold many things. A plate can be placed on a table if need be, and it also can hold other items such as food, etc."